package.json

```diff
@@ -78,8 +82,8 @@
     "karma-mocha": "^1.3.0",
     "karma-phantomjs-launcher": "^1.0.4",
     "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^3.0.0",
-    "lodash": "^4.17.11",
+    "karma-webpack": "^3.0.5",
+    "lodash": "^4.17.15",
     "mocha": "^5.1.1",
     "node-sass": "^4.10.0",
```

Comment:
My system-wide node was upgraded to node-12 and node-sass stopped working. I think it would make sense to add "node@lts" to the dependencies to get rid of problems with the system node.

Reply:
I have added node to the dev dependencies